azure-devops/provision-image.ps1

```diff
@@ -201,8 +179,4 @@ $environmentKey = Get-ItemProperty -Path 'HKLM:\SYSTEM\CurrentControlSet\Control
 Set-ItemProperty -Path 'HKLM:\SYSTEM\CurrentControlSet\Control\Session Manager\Environment' `
     -Name Path `
     -Value "$($environmentKey.Path);C:\Program Files\CMake\bin;C:\Program Files\LLVM\bin"
-InstallZip 'Azure DevOps Agent' $VstsAgentUrl 'C:\agent'
-Add-MpPreference -ExclusionPath C:\agent

```

Comment:
does this still happen at some point? or do we just never enable defender on the generated images

Reply:
We don't turn Defender off any more, that's something we'll need to talk to Azure Pipelines team about since we no longer know where the work tree is.

And actually this was broken before since it should have excluded `D:\` after we moved the build there.